src/Pages/MyungJunKim/Main/Contents.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import ContentHeader from '../Main/ContentHeader';
+import ContentImage from '../Main/ContentImage';
+import CommentBox from '../Main/CommentBox';
+
+class Contents extends Component {
+  constructor() {
+    super();
+    this.state = {
+      inputValue: '',
+      commentList: [],
+      likeState: true,
+      isBtnLike: 'fas fa-heart btn-empty',
+      isCommentModal: false,
+    };
+  }
+
+  handleCommentModal = e => {
```

Comment:
- e 는 사용하지 않는데, 인자로 넘겨줄 필요가 없을 것 같습니다!
```suggestion
  handleCommentModal = () => {
```

Reply:
넵! 수정했습니다!